### PR TITLE
first french version

### PR DESCRIPTION
**Pull request by romainBechet**
First version, 
Some checks to do (Where each json object is placed) and a reread.

Same issue as the Spanish version for the `cohend.json` at `gatsby-theme-rpsych-cohend/i18n/react-i18next/es`, 
I copy/paste his message 
The phrase: "Interpreting Cohen's" (omitting the d) can not be directly translated to French, since the French equivalent to the English possessive ([NOUN]’s) is placed before the noun.

### CODE REVIEW

File: gatsby-theme-rpsych-cohend/content/cohendTranslators.yml [hunk 1]
```diff
@@ -1,16 +1,11 @@
 - lang: en
 - lang: se
+- lang: fr
   translator: 
     -
```
Comment (rpsychologist) [inline gatsby-theme-rpsych-cohend/content/cohendTranslators.yml]: new component, add your info here if you want to be acknowledged

File: gatsby-theme-rpsych-cohend/i18n/react-i18next/fr/cohend.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "Interpreting Cohen's": "Interpreting Cohen's",
   "effect size": "taille d'effet",
-  "Cohen's": "Cohen",
   "Cohen's d": "d de Cohen",
+  "CohenLabelSlider": "<i>d</i> de Cohen",
+  "CohenU3": "U<1>3</1> de Cohen",
```
Comment (rpsychologist) [inline gatsby-theme-rpsych-cohend/i18n/react-i18next/fr/cohend.json:5]: new keys, is this correct?

File: gatsby-theme-rpsych/i18n/config.json [hunk 3]
```diff
@@ -14,5 +14,13 @@
     "localName": "Svenska",
     "langDir": "ltr",
     "dateFormat": "YYYY.MM.DD"
+  },
+  {
+    "code": "fr",
+    "hrefLang": "fr",
+    "name": "French",
+    "localName": "Français",
```
Comment (rpsychologist) [inline gatsby-theme-rpsych/i18n/config.json:22]: correct localName?

File: gatsby-theme-rpsych/i18n/react-i18next/fr/blog.json [hunk 4]
```diff
@@ -6,9 +6,8 @@
   "GitHubSponsors": "Vous pouvez sponsoriser mon travail open source en utilisant les <2>Sponsors GitHub</2> et avoir votre nom ici.",
   "Webmentions sent before 2021": "(Les Webmentions envoyées avant 2021 ne seront malheureusement plus affichées ici.)",
   "Designed and built by": "Designé et développé par Kristoffer Magnusson. Hébergé par",
-  "bought": "a acheté",
-  "coffee": "café",
-  "coffees": "cafés",
+  "coffeeSupporters": "<0>{{name}}</0> a acheté {{coffeeSymbols}} ({{count}}) café",
+  "coffeeSupporters_plural": "<0>{{name}}</0> a acheté {{coffeeSymbols}} ({{count}}) cafés",
```
Comment (rpsychologist) [inline gatsby-theme-rpsych/i18n/react-i18next/fr/blog.json:10]: correct?

File: gatsby-theme-rpsych/i18n/react-i18next/fr/blog.json [hunk 5]
```diff
@@ -18,6 +17,7 @@
   "Save settings in browser": "Sauvegarder les paramètres du navigateur",
   "Clear": "Effacer",
   "authorBio": "Ecrit par <strong>Kristoffer Magnusson</strong>, un chercheur en psychologie clinique. <4>Vous pouvez le suivre sur Twitter</4> et venir sur le Discord <8>Git Gud Science</8>.",
+  "Translated by": "Traduit par",
```
Comment (rpsychologist) [inline gatsby-theme-rpsych/i18n/react-i18next/fr/blog.json:20]: correct?

**Review by rpsychologist — COMMENTED** [review]
I've added some minor tweaks to get it ready for production, are the marked translations OK?

File: gatsby-theme-rpsych-cohend/i18n/react-i18next/fr/cohend.json (hunk 2 above)
Comment (romainBechet) [reply gatsby-theme-rpsych-cohend/i18n/react-i18next/fr/cohend.json:5]: perfect!

File: gatsby-theme-rpsych/i18n/config.json (hunk 3 above)
Comment (romainBechet) [reply gatsby-theme-rpsych/i18n/config.json:22]: Yes

File: gatsby-theme-rpsych/i18n/react-i18next/fr/blog.json (hunk 4 above)
Comment (romainBechet) [reply gatsby-theme-rpsych/i18n/react-i18next/fr/blog.json:10]: Yes

File: gatsby-theme-rpsych/i18n/react-i18next/fr/blog.json (hunk 5 above)
Comment (romainBechet) [reply gatsby-theme-rpsych/i18n/react-i18next/fr/blog.json:20]: Correct

**Pull request merged**